src/Core/IIssuesFilter.cs

```diff
@@ -19,12 +19,25 @@
  */
 
 using System.Collections.Generic;
-using Sonarlint;
 
-namespace SonarLint.VisualStudio.Integration.Vsix.SonarLintTagger
+namespace SonarLint.VisualStudio.Core
 {
+    /// <summary>
+    /// Describes a single issue with the properties required for
+    /// it to be compared against server-side issues by the issues filter
+    /// </summary>
+    public interface IFilterableIssue
```

Comment:
The daemon Protobuf `Issue` class doesn't contain enough information to be able to compare against server-side issues. Specifically, it's missing the `WholeLineText` and `LineHash` properties.

I haven't added any implementations of this interface yet; I'll do that in the next commit.